src/routes/init/index.ts

```diff
@@ -1,20 +1,23 @@
 import { Request } from 'express';
 import { DbClient, IDbClient } from '../../dal/interface';
 import { InsertWriteOpResult } from 'mongodb';
-import { data } from './output';
+import { importDbEntries } from './import';
 import { applyTracing } from '../../utils/tracer/index';
 import { CSEResponse } from '../../server/index';
+import { IClaim } from '../../models/claim/index';
 
 
 export const handler = async (req: Request, res: CSEResponse): Promise<void> => {
+    const output: IClaim[] = await importDbEntries();
```

Comment:
We're going to need to add a `try/catch` around everything that is preceded by the `await` keyword for error handling.